ci/build_libtelio.py

```diff
@@ -202,6 +202,13 @@ def copy_bindings(config):
 
 
 def main() -> None:
+    uniffi_test_bindings = False 
+    for arg in sys.argv:
+        if arg == "uniffi-test-bindings":
+            uniffi_test_bindings = True 
+            sys.argv.remove("uniffi-test-bindings")
+            break
+
```

Comment:
why do we manually process the command options - can this be converted to the python built in argparse (like the code right below) ?